tests/download_noaa_aws.sh

```diff
@@ -87,7 +104,6 @@ elif [[ "$satellite" == "geokompsat" ]]; then
     echo "rclone --config $rclone_conf lsf $rclone_path"
     files=`rclone --config $rclone_conf lsf $rclone_path`
     echo "COMPARE: ${yyyy}${mm}${dd}${hh}${mn}"
-    break
```

Comment:
I don't think this was supposed to be here under the geokompsat section

Reply:
I do not think so either! Resolving this conversation